lib/src/features/main/theme/theme_data/app_theme_data.dart

```diff
@@ -1,283 +1,62 @@
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w700,
-      fontSize: 24,
-      height: 36 / 24,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    titleLarge: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w300,
-      fontSize: 20,
-      height: 28 / 20,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    titleMedium: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w400,
-      fontSize: 16,
-      height: 24 / 16,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    titleSmall: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w400,
-      fontSize: 12,
-      height: 16 / 12,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    bodyLarge: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w300,
-      fontSize: 16,
-      height: 24 / 16,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    bodyMedium: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w400,
-      fontSize: 14,
-      height: 20 / 14,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    bodySmall: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w400,
-      fontSize: 12,
-      height: 18 / 12,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    labelLarge: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w400,
-      fontSize: 16,
-      height: 20 / 16,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    labelMedium: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w500,
-      fontSize: 14,
-      height: 20 / 14,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-    labelSmall: TextStyle(
-      fontFamily: FontFamily.inter,
-      color: SurfacePallet.light.surface3,
-      fontStyle: FontStyle.normal,
-      fontWeight: FontWeight.w500,
-      fontSize: 10,
-      height: 16 / 10,
-      leadingDistribution: TextLeadingDistribution.even,
-    ),
-  );
+  // TODO(Esmaeil): Update this part based on the new text style in the Figma.
```

Comment:
@PouriaMoradi021  , We need to add text style details here, but access to Figma's Dev Mode is required. Since this may take time, we'll defer it and handle it in a separate issue.

Reply:
> @PouriaMoradi021 , We need to add text style details here, but access to Figma's Dev Mode is required. Since this may take time, we'll defer it and handle it in a separate issue.

I agree, @esmaeil-ahmadipour . I suggest that this be done by myself in a separate issue after granting the desired access in Figma.

Reply:
this item explained in this [commit](https://github.com/pactus-project/pactus-gui/issues/45#issuecomment-2563754458)  and will be resolve in the issue #45 .